creator/files/migrations/0024_set_old_types_to_oth.py

```diff
@@ -0,0 +1,24 @@
+from django.db import migrations
```

Comment:
Could you please combine the two migrations into one?

Reply:
I read that using postgres you should isolate the migration using `RunPython`:

https://docs.djangoproject.com/en/3.2/ref/migration-operations/#django.db.migrations.operations.RunPython

```
Thus, on PostgreSQL, for example, you should avoid combining schema changes and RunPython operations in the same migration or you may hit errors like OperationalError: cannot ALTER TABLE "mytable" because it has pending trigger events.
```

Reply:
Oh I see